pulsar/win_notify/pillar/beacons.sls

```diff
@@ -0,0 +1,6 @@
+beacons:
+  win_notify:
+    C:/Users:
+
+  interval: 300 # Do not change.  Hard coded into win_notify
```

Comment:
why define the setting if it's hard-coded? if this is a TODO, let's leave it out here but add it to our roadmap.